ivy/functional/frontends/tensorflow/random.py

```diff
@@ -20,11 +20,18 @@ def uniform(shape, minval=0, maxval=None, dtype=ivy.float32, seed=None, name=Non
 def normal(shape, mean=0.0, stddev=1.0, dtype=ivy.float32, seed=None, name=None):
     return ivy.random_normal(mean=mean, std=stddev, shape=shape, dtype=dtype, seed=seed)
 
-
 # implement random shuffle
 @with_unsupported_dtypes(
     {"2.9.0 and below": ("int8", "int16", "in32", "int64", "unsigned")}, "tensorflow"
 )
 @to_ivy_arrays_and_back
 def shuffle(value, seed=None, name=None):
     return ivy.shuffle(value, seed=seed)
+
+@to_ivy_arrays_and_back
+def stateless_uniform(shape, seed, minval=0, maxval=None, dtype=ivy.float32, name=None, alg='auto_select'):
+    seed = seed[0]
```

Comment:
That line sounds redundant, why don't you just send `seed[0]` to `ivy.random_uniform`

Reply:
Thanks. I just did it. And I decided to pass `seed[0]+seed[1]` so that both elements in the `seed` can be utilized.

Reply:
Awesome, lgtm